assets/src/edit-story/components/mediaPicker/useMediaPicker.js

```diff
@@ -19,28 +19,21 @@
  */
 import { useEffect } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
-/**
- * External dependencies
- */
-import PropTypes from 'prop-types';
-import styled from 'styled-components';
 
-function UploadButton({
-  title,
-  buttonText,
-  buttonCSS,
-  buttonInsertText,
-  multiple,
-  onSelect,
-  onClose,
-  type,
+export default function useMediaPicker({
```

Comment:
This is being refactored, you may want to look at [this](https://github.com/WordPress/gutenberg/blob/dbcaf38e886781d0bf5101db8cd04b5360ce2685/packages/media-utils/src/components/media-upload/index.js).

Reply:
Hey @spacedmonkey can you elaborate on this?